spec/requests/users_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```

Comment:
- [ ] I cannot find any relevant spec tests for the user index and user show requests. This file should contain your posts tests, rather it contains dummy pending tests generated with Rails. Kindly add specs here for index and show requests.

- [ ] Kindly ensure you cover the required scenarios, like rendering template, checking for placeholder, and maybe HTTP status.